src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -461,6 +467,32 @@ private void verifyUpload(Exporter3Configuration ex3Config, String uploadId, Str
 
         long participantModifiedOn = Long.parseLong(rowMap.get("modifiedOn"));
         assertTrue(participantModifiedOn > oneHourAgo.getMillis());
+
+        // Verify the record in Bridge.
+        ForWorkersApi workersApi = adminDeveloperWorker.getClient(ForWorkersApi.class);
+        HealthDataRecordEx3 record = workersApi.getRecordEx3(IntegTestUtils.TEST_APP_ID, uploadId).execute().body();
+        assertTrue(record.isExported());
+        assertTrue(record.getExportedOn().isAfter(oneHourAgo));
+
+        // Verify the presigned download url for a health record is generated and contains the data expected.
+        String hostUrl = user.getClientManager().getHostUrl();
+        String url = "/v3/participants/self/exporter3/healthdata/" + uploadId + "?download=true";
```

Comment:
There's no need to make a raw HTTP request. You can use ForConsentedUsersApi and use the download flag that you added in JavaSdk. (You can get ForConsentedUsersApi from the user. There should be an example elsewhere in this test class.)

While we're at it, we can also test that expiration is set and is sometime in the future. (Maybe check it's at least 30 minutes in the future, to avoid clock skew issues.)

In fact, you could probably replace the call to ForWorkersApi entirely with this.